unifi-linuxserver.json

```
@@ -83,13 +83,13 @@
             }
         },
         "description": "Unifi Access Point controller",
```

Add docker image info and link:
```suggestion
        "description": "Unifi Access Point controller. <p>Based on the LinuxServer.io Unifi-controller docker image: <a href='https://hub.docker.com/r/linuxserver/unifi-controller' target='_blank'>https://hub.docker.com/r/linuxserver/unifi-controller</a>.",
```